App/index.js

```diff
@@ -0,0 +1,142 @@
+const verde = document.getElementById('verde');
+const rojo = document.getElementById('rojo');
+const amarillo = document.getElementById('amarillo');
+const azul = document.getElementById('azul');
+
+const btnIniciar = document.getElementById('btnIniciar');
+
+const ULTIMO_NIVEL = 10;
+class Juego {
+  constructor() {
+    this.iniciarJuego = this.iniciarJuego.bind(this);
+    this.iniciarJuego();
+    this.generarSecuencia();
+    setTimeout(this.pasarDeNivel, 700);
+  }
+
+  iniciarJuego() {
+    this.elegirColor = this.elegirColor.bind(this);
+    this.pasarDeNivel = this.pasarDeNivel.bind(this);
+    this.toggleBtnIniciar();
+    this.nivel = 1;
```

Comment:
Podrías agregar una variable para setear el nivel inicial

```suggestion
    this.nivel = INITIAL_LEVEL;
```

Reply:
Listo pero para seguir con el código en español, la nombre NIVEL_INICIAL